app/helpers/application_helper.rb

```diff
@@ -22,4 +22,9 @@ def ethnicity_options
       ['Middle Eastern', 4]
     ], get_pre_selected_option("ethnicity", nil))
   end
+  
+  def age(dob)
+    now = Time.now.utc.to_date
+    now.year - dob.year - ((now.month > dob.month || (now.month == dob.month && now.day >= dob.day)) ? 0 : 1)
+  end
```

Comment:
Yes it depends on accuracy. I think it's probably good enough to use year. See also how rails went about this: http://api.rubyonrails.org/classes/ActionView/Helpers/DateHelper.html#method-i-time_ago_in_words